### PR TITLE
added some features tor, databooks, webui

### PR DESCRIPTION
**Pull request by lucmski**

### CODE REVIEW

**Comment by tdh8316** [comment]
I appreciate your contributions!

But before merging these changes, I have some request.

---

First, This PR contains too many changes, can confuse.
I'm not an expert in the Go language. Accordingly, I cannot understand perfectly what these changes mean.
It would be nice if you make a pull request which only contains TOR support, for selective acceptance.

Honestly, I couldn't comprehend what databooks and WebUI are.
My bad 😭 

---

Second, you should discuss with [Sherlock Team](https://github.com/sherlock-project/sherlock) to improve the database because the database of Investigo is same as Sherlock's.

---

Thanks for your contribution again!

File: main.go [hunk 1]
```diff
@@ -0,0 +1,663 @@
...
+	"github.com/kamilsk/retry/strategy"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/qor/admin"
+	"golang.org/x/net/proxy"
+	// "github.com/spf13/cobra"
+)
+
+/*
+Similar:
+- https://github.com/mesuutt/sherlock/blob/master/main.go
```
Comment (tdh8316) [inline main.go:33]: Lol... I never agree with this source code is similar to that.

**Review by tdh8316 — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (lucmski) [reply main.go:33]: do you have a telegram account for discussing ?

Comment (tdh8316) [reply main.go:33]: Yes, I have an account for telegram: @tdh8316.

**Pull request closed without merging**